src/components/TokenAdministrative.js

```diff
@@ -98,15 +99,37 @@ class TokenAdministrative extends React.Component {
    * Update token state after didmount or props update
    */
   updateWalletInfo = () => {
-    const mintUtxos = this.props.wallet.getMintAuthority(this.props.token.uid, { many: true });
-    const mintCount = mintUtxos ? mintUtxos.length : 0;
+    if (!this.props.token && !this.props.token.uid) {
```

Comment:
Should this be an `||` operator?

Reply:
Nice catch, thanks

Fixed in a7a7e5b343ff8b575c714dfa04f830920b3ae6f3